### DISCUSSION TITLE
private commits does not show even though we add count_private prams in the url

### DISCUSSION THREAD

**Discussion by gokul-369** [anuraghazra/github-readme-stats#3094] [Q&A] [answered]
### Describe the bug

"https://github-readme-stats.vercel.app/api?username=gokul-369&show_icons=true&locale=en&theme=tokyonight&hide_border=true&count_private=true"

this is the url that i have used and yet it does not count the commits that i made to my private repo

### Expected behavior

it should count the private contributions too

**Comment by rickstaa** [comment, marked as answer]
@gokul-369 the `include_private` attribute has been deprecated since it did not count the commits but all contributions (see https://github.com/anuraghazra/github-readme-stats/pull/2736). To get the private stats to show up, you should [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own).
